Sources/TensorFlow/Loss.swift

```diff
@@ -86,6 +86,26 @@ public func categoricalHingeLoss<Scalar: TensorFlowFloatingPoint>(
     return max(Tensor(0), negative - positive + Tensor(1))
 }
 
+// Helper function for Logcosh
+@differentiable(wrt: x)
+fileprivate func logCosh<Scalar: TensorFlowFloatingPoint>(
+    x: Tensor<Scalar>
+) -> Tensor<Scalar> {
+    return  x + softplus(Tensor(-2) * x) - log(Tensor(y))
```

Comment:
Please remove the extra space between `return` and the expression.  To take this a step further, you can omit the `return` keyword because the function is a one-liner.
```suggestion
    x + softplus(Tensor(-2) * x) - log(Tensor(y))
```